docker/conf/log.config

```diff
@@ -49,11 +49,13 @@ handlers:
     class: logging.StreamHandler
     formatter: precise
 
+{% if not SYNAPSE_LOG_SENSITIVE %}
 loggers:
     synapse.storage.SQL:
         # beware: increasing this to DEBUG will make synapse log sensitive
         # information such as access tokens.
         level: INFO
+{% endif %}
```

Comment:
I'm confused about which way around the condition ought to be.
An new comment explaining what the purpose of this block is would be useful.